js/content-filters.js

```diff
@@ -0,0 +1,78 @@
+import {debounce} from './functions.js';
+
+const FiltersId = {
+  DEFFAULT: '#filter-default',
```

Comment:
```suggestion
  DEFAULT: '#filter-default',
```

Каким IDE ты пользуешься, VSCode? Есть расширение, которое подсвечивает такие опечатки https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker